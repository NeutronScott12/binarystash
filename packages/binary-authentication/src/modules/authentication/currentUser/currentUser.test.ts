import { createTestClient } from 'apollo-server-testing';
import { gql } from 'apollo-server-express';

import server from '../../..';

describe('Current User', () => {
	test('Should return invalid credentials', async () => {
		const { query } = createTestClient(server);

		const res = await query({
			query: gql`
				{
					Query {
						currentUser {
							id
							email
						}
					}
				}
			`
		});

		expect(res).toBe({});
	});
});
