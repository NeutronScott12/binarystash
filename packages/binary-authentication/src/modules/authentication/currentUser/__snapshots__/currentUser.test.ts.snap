// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Current User Should return invalid credentials 1`] = `
Object {
  "data": undefined,
  "errors": Array [
    [ValidationError: Cannot query field "Query" on type "Query". Did you mean "queryUsers"?],
  ],
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;
